docs/cep/table/index.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 1
+title: Table (Collection)
+---
+
+A table is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../collections/index.md).
+
+## Purpose
+
+Tables allow the stream worker to work with stored events. By defining a schema for tables, the stream processor enables them to be processed by queries using their defined attributes with the streaming data. You can also query the state of the stored events in the table.
+
+Any table defined in a stream worker is automatically a store for that stream worker. Tables can be stores or [sinks](../sink/).
+
+You can query table contents using [Table Operators](table-operators).
+
+## Syntax
+
+There are two ways to define tables.
+
+### CREATE TABLE
+
+The syntax for a new table definition is as follows:
+
+```sql
+CREATE TABLE (GLOBAL|SPOT)? <table_name> (<attribute_name> <attribute_type>, ...);
```

Comment:
@diana-macrometa - The `SPOT` type is more or less deprecated but the local is missing, should (GLOBAL | LOCAL), the default is `GLOBAL`.